.github/workflows/release.yml

```diff
@@ -172,8 +164,6 @@ jobs:
         if: runner.os == 'Linux'
         run: |
           mkdir archive_aarch64
-          cp target/aarch64-unknown-linux-gnu/release/dora-coordinator archive_aarch64
-          cp target/aarch64-unknown-linux-gnu/release/dora-daemon archive_aarch64
           cp target/aarch64-unknown-linux-gnu/release/dora archive_aarch64/dora
           cd archive_aarch64
           zip -r ../archive_aarch64.zip .
```

Comment:
As we only distribute a single binary as artifact, we don't need to zip it. Or is there any advantage in zipping it?